scipy/interpolate/interpnd.pyx

```diff
@@ -929,17 +982,18 @@ class CloughTocher2DInterpolator(NDInterpolatorBase):
         cdef const double_or_complex[:,:,:] grad = self.grad
         cdef double_or_complex[:,::1] out
         cdef const int[:,::1] simplices = self.tri.simplices
-        cdef double c[NPY_MAXDIMS]
+        cdef double[:,:] c
```

Comment:
I wonder if the original was a perf optimization. At any rate, we'd need a benchmark to make sure there is no performance regression (C array -> cython memoryview, python functions in `__call__` etc).

Reply:
Since all the changed code is used by `griddata` with different methods, and `griddata` already has a benchmark, is it necessary to add another one?
I am adding a benchmark for the subclass anyway, same as in RGI.